main.py

```diff
@@ -79,7 +79,30 @@ async def on_message(message: Message):
             await message.channel.send("You need to use !list first, sugar.")
             return
 
-        await command_play(message)
+        # split on spaces
+        # TODO: Possibly parse arguments before casing on command
```

Comment:
What do you mean by this TODO?

Reply:
Ideally arguments would be split and parsed before this specific condition, but I guess there's no extra considerations necessary at the moment.